core/dataset_manager.py

```diff
@@ -200,11 +195,9 @@ def _generate_ed_directory(self):
     def post_directories_and_category_labels(self, key):
         """
         Post the ED Directory on blockchain with the given key.
-
         The ED Directory is a JSON dictionary whose keys represent the dataset
         folders in the directory and whose values represent the corresponding
         datasets. Assume only one dataset file per folder. 
-
```

Comment:
Why all the line deletions? The lines were put to separate different parts of each docstring (args vs return vs example vs conclusion vs etc.), but now the docstrings are harder to read.

Reply:
I tried to undo all of this by copy-pasting the dataset_manager.py file back in from Master...let me fix it

Reply:
When I'm thinking I just randomly delete and add whitespace